src/Client.php

```diff
@@ -272,7 +273,126 @@ public function getAsset($assetId) {
       ['headers' => $this->getDefaultHeaders()]
     );
 
-    return (string) $response->getBody();
+    $asset = Asset::fromJson((string) $response->getBody());
```

Comment:
I'm gonna actually split this out into a separate thing. I'll credit you on the commit.